react/Footer/Footer.common.less

```diff
@@ -0,0 +1,25 @@
+@import (reference) "~seek-style-guide/theme";
+
+@footer-vertical: ~"and (max-width: 767px)";
```

Comment:
Correct, this is centralising what already exists for all the candidate products (excluding CR). Iterating on the responsive design once in the style guide will address these break points.